transformations/tests/test_cloudsc.py

```diff
@@ -27,54 +25,32 @@ def fixture_here():
 
 
 @pytest.fixture(scope='module', name='local_loki_bundle')
-def fixture_local_loki_bundle(here):
+def fixture_local_loki_bundle():
     """Inject ourselves into the CLOUDSC bundle"""
-    lokidir = Path(__file__).parent.parent.parent
-    target = here/'source/loki'
-    backup = here/'source/loki.bak'
-    bundlefile = here/'bundle.yml'
-    local_loki_bundlefile = here/'__bundle_loki.yml'
-
-    # Do not overwrite any existing Loki copy
-    if target.exists():
-        if backup.exists():
-            shutil.rmtree(backup)
-        shutil.move(target, backup)
-
-    # Change bundle to symlink for Loki
-    bundle = yaml.safe_load(bundlefile.read_text())
-    loki_index = [i for i, p in enumerate(bundle['projects']) if 'loki' in p]
-    assert len(loki_index) == 1
-    if 'git' in bundle['projects'][loki_index[0]]['loki']:
-        del bundle['projects'][loki_index[0]]['loki']['git']
-    bundle['projects'][loki_index[0]]['loki']['dir'] = str(lokidir.resolve())
-    local_loki_bundlefile.write_text(yaml.dump(bundle))
-
-    yield local_loki_bundlefile
-
-    if local_loki_bundlefile.exists():
-        local_loki_bundlefile.unlink()
-    if target.is_symlink():
-        target.unlink()
-    if not target.exists() and backup.exists():
-        shutil.move(backup, target)
+    return _local_loki_bundle
 
 
 @pytest.fixture(scope='module', name='bundle_create')
 def fixture_bundle_create(here, local_loki_bundle):
     # Run ecbundle to fetch dependencies
+    bundle = str(next(local_loki_bundle(here)))
     execute(
-        ['./cloudsc-bundle', 'create', '--bundle', str(local_loki_bundle)],
+        ['./cloudsc-bundle', 'create', '--bundle', bundle],
         cwd=here,
         silent=False
     )
 
 
+@pytest.fixture(scope='module', name='write_script')
+def fixture_write_script():
+    return _write_script
+
+
 @pytest.mark.usefixtures('bundle_create')
 @pytest.mark.parametrize('frontend', available_frontends(
     skip=[(OMNI, 'OMNI needs FParser for parsing dependencies')] if not HAVE_FP else None
 ))
-def test_cloudsc(here, frontend):
+def test_cloudsc(here, write_script, frontend):
```

Comment:
`write_script` doesn't need to be a fixture, that can just be a regular function call.